### DISCUSSION TITLE
[A-L] (2023/24) Foglio 10 - Esercizio 5

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#215] [Esercizi - Bertini] [answered]
![immagine](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/9353028c-50d0-4b3c-aa4b-c1fd21ab58fb)

![immagine](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/9c5c8efb-03ba-4192-a64e-9149f874a995)

**Comment by Elia-Belli** [comment, marked as answer]
## Soluzione confermata in classe
**(1)** Se $X$ conta il numero di volte in cui otteniamo doppio uno allora si tratta di un'esperimento binario con risultati "ottengo doppio uno" o "non ottengo doppio 1" ripetuto $300$ volte in cui ogni volta la probabilità di ottenere doppio uno è $p=1/36$, quindi:
$$X\sim Bin(300,1/36)$$

$$\mathbb{E}(X)=np=\frac{300}{36}=\frac{25}{3}\simeq 8.3$$

$$\mathbb{V}(X)=np(1-p)=\frac{300\cdot 35}{36^2}=\frac{875}{108}\simeq 8.1$$

**(2)** Approssimando $X$ con una gaussiana poniamo $\mu=25/3$ e $\sigma^2=875/108$, consideriamo quindi $X\sim\mathcal{W}(25/3,875/108)$ e riconduciamo ad una gaussiana standard:
$$Z=(X-\frac{25}{3})\cdot \frac{36}{10\sqrt{105}}\to X=\frac{10\sqrt{105}}{36}Z+\frac{25}{3}$$

$$\mathbb{P}(X\gt 10)=\mathbb{P}(Z>\frac{6}{\sqrt{105}})=1-\mathbb{P}(Z\leq \frac{6}{\sqrt{105}})\simeq 1-\mathbb{P}(Z\leq 0.59)=1-0.7224=0.2776$$

**(3)** Troviamo $n$ affinchè $\mathbb{P}(X\geq 10)>1/2$, consideriamo l'approssimazione gaussiana $V\sim\mathcal{W}(n/36,n\cdot 35/36^2)$ e normalizziamo:

$$Z=\frac{V-\frac{n}{36}}{\frac{\sqrt{35n}}{36}}\to V= \frac{Z\sqrt{35n}}{36}+\frac{n}{36}$$

Effettuiamo la sostituzione e imponiamo il vincolo:

$$\mathbb{P}(V\geq 10)=\mathbb{P}(Z\geq \frac{(10-\frac{n}{36})36}{\sqrt{35n}})=1-\mathbb{P}(Z\lt \frac{(10-\frac{n}{36})36}{\sqrt{35n}})\gt \frac{1}{2}$$

$$\Rightarrow \frac{(10-\frac{n}{36})36}{\sqrt{35n}}=0\to n=360$$

**Comment by Warcophyr** [comment]
![image](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/133346488/8332da09-06e4-4e77-b9a5-aefd893e73ef)
![image](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/133346488/b56f5a14-5d40-4b24-a0f6-74dffeba396b)
![image](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/133346488/6fa4f31e-1247-47d6-9062-047dc524b1e1)
![image](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/133346488/1489ca7d-5ad6-4940-b663-158e2bc68102)
![image](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/133346488/d577620a-0d56-4c24-a43f-907b375955b5)
![image](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/133346488/6d56b42f-6094-4058-a6bc-8383722ef273)
